platform/platform-app/src/main/resources/features.xml

```diff
@@ -808,6 +808,7 @@
         <feature prerequisite="true" version="${karaf.version}">eventadmin</feature>
         <feature prerequisite="true" version="${karaf.version}">jasypt-encryption</feature>
         <feature prerequisite="true" version="${karaf.version}">http</feature>
+        <bundle>mvn:ddf.lib/common-system/${project.version}</bundle>
```

Comment:
This seems like a weird place to put this. This feature currently holds Karaf-specific stuff.